tests/system/t7894.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-disabled: true
+    disabled: true
```

Comment:
Created around 2018-08 in a commit that references #7894 and #8496
 Disabled on x32 around 2019-02 due to flakiness on CI - references #10608
 Seems to run fine. Re-enable?

Reply:
This is a silly brute force test that'll make CI grumpy.

Should be a way to launch the test with limited memory to see if we can trigger the failure consistently.

Reply:
Keeping this disabled for now.